### DISCUSSION TITLE
I don't care about activity of people I sponsor

### DISCUSSION THREAD

**Discussion by r-thomson** [community/community#88420] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

With my feed set to _only_ the "Stars" event type, I'm seeing "____ starred a repository" events from people I'm not following. After clicking the "⋯" button, it says "You're seeing this because you sponsor ____".

I don't want to see this information in the feed, it's just noise. If I wanted to see that person's activity, then I would have _followed_ them.

**Comment by c0m4r** [comment]
+1, ultra annoying
